graph/pkg/service/v0/graph_test.go

```diff
@@ -161,29 +162,93 @@ var _ = Describe("Graph", func() {
+				Status: status.NewOK(ctx),
+				StorageSpaces: []*provider.StorageSpace{
+					{
+						Id:        &provider.StorageSpaceId{OpaqueId: "aID!differentID"},
+						SpaceType: "mountpoint",
+						Root: &provider.ResourceId{
+							StorageId: "aID",
+							OpaqueId:  "differentID",
+						},
+						Name: "New Folder",
+						Opaque: &typesv1beta1.Opaque{
+							Map: map[string]*typesv1beta1.OpaqueEntry{
+								"spaceAlias":     {Decoder: "plain", Value: []byte("mountpoint/new-folder")},
+								"etag":           {Decoder: "plain", Value: []byte("101112131415")},
+								"grantStorageID": {Decoder: "plain", Value: []byte("ownerStorageID")},
+								"grantOpaqueID":  {Decoder: "plain", Value: []byte("opaqueID")},
+							},
+						},
+					},
+				},
+			}, nil)
+			gatewayClient.On("Stat", mock.Anything, mock.Anything).Return(&provider.StatResponse{
+				Status: status.NewOK(ctx),
+				Info: &provider.ResourceInfo{
+					Etag:  "123456789",
+					Type:  provider.ResourceType_RESOURCE_TYPE_CONTAINER,
+					Id:    &provider.ResourceId{StorageId: "ownerStorageID", OpaqueId: "opaqueID"},
+					Path:  "New Folder",
+					Mtime: &typesv1beta1.Timestamp{Seconds: 1648327606},
+					Size:  uint64(1234),
+				},
+			}, nil)
+			gatewayClient.On("GetQuota", mock.Anything, mock.Anything).Return(&provider.GetQuotaResponse{
+				Status: status.NewUnimplemented(ctx, fmt.Errorf("not supported"), "not supported"),
+			}, nil)
+
+			r := httptest.NewRequest(http.MethodGet, "/graph/v1.0/me/drives", nil)
+			rr := httptest.NewRecorder()
+			svc.GetDrives(rr, r)
+
+			Expect(rr.Code).To(Equal(http.StatusOK))
+
+			body, _ := io.ReadAll(rr.Body)
+
+			var response map[string][]libregraph.Drive
+			err := json.Unmarshal(body, &response)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(*response["value"][0].DriveAlias).To(Equal("mountpoint/new-folder"))
+			Expect(*response["value"][0].DriveType).To(Equal("mountpoint"))
+			Expect(*response["value"][0].Id).To(Equal("aID!differentID"))
+			Expect(*response["value"][0].Name).To(Equal("New Folder"))
+			Expect(*response["value"][0].Root.WebDavUrl).To(Equal("https://localhost:9200/dav/spaces/aID!differentID"))
+			Expect(*response["value"][0].Root.ETag).To(Equal("101112131415"))
+			Expect(*response["value"][0].Root.Id).To(Equal("aID!differentID"))
+			Expect(*response["value"][0].Root.RemoteItem.ETag).To(Equal("123456789"))
+			Expect(*response["value"][0].Root.RemoteItem.Id).To(Equal("ownerStorageID!opaqueID"))
+			Expect(*response["value"][0].Root.RemoteItem.LastModifiedDateTime).To(Equal(time.Unix(1648327606, 0)))
+			Expect(*response["value"][0].Root.RemoteItem.Folder).To(Equal(libregraph.Folder{}))
+			Expect(*response["value"][0].Root.RemoteItem.Name).To(Equal("New Folder"))
+			Expect(*response["value"][0].Root.RemoteItem.Size).To(Equal(int64(1234)))
+			Expect(*response["value"][0].Root.RemoteItem.WebDavUrl).To(Equal("https://localhost:9200/dav/spaces/ownerStorageID!opaqueID"))
```

Comment:
1) Would look much nicer if we would assign `v := *response["value"][0]` and later `Expect(v.Something)...` in all cases. 

2) We could do `Expect(len(*response["value"])).To(Equal(1))` to avoid panics while the testsuite runs

Both not critical since it's only a test.